manifests/4.3/logforwardings.crd.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: clusterloggings.logging.openshift.io
+spec:
+  group: logging.openshift.io
+  names:
+    kind: LogForwarding
+    listKind: LogForwardingList
+    plural: logforwardings
+    singular: logforwarding
```

Comment:
@bparees In the interest of reserving the ability to have namespaced logforwarding in the future, should the kind maybe be `clusterlogforwarding`

Reply:
that seems reasonable.  though in that case you could also make it a clusterscoped resource (is clusterlogging a cluster scoped resource today?)

Reply:
no.

Reply:
I guess hedge it by naming it clusterlogforwarding for now and we can reevaluate before we create/name the GA api.

Reply:
deferring to a GA discussion